deploy/helm/sumologic/values.yaml

```diff
@@ -181,6 +187,15 @@ sumologic:
     ## ref: https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter#configuration
     bearerCacheTtl: "3600"
 
+
+## Configure metrics-server
+## ref: https://github.com/helm/charts/blob/master/stable/metrics-server/values.yaml
+metrics-server:
+  enabled: true
+  args:
+    - --kubelet-insecure-tls
```

Comment:
I could see customers having an issue with using this in an insecure way.  Is there any other way to run this?

Reply:
Without using that the metrics server is unable to get metrics.  Will look into it more if we can avoid this.
https://github.com/kubernetes-sigs/metrics-server/issues/131